src/autoswagger.ts

```diff
@@ -58,15 +60,14 @@ export class AutoSwagger {
 				<script>
 						window.onload = function() {
 							SwaggerUIBundle({
-								url: "` +
-        url +
-        `",
+								url: "` + url + `",
 								dom_id: '#swagger-ui',
 								presets: [
 									SwaggerUIBundle.presets.apis,
 									SwaggerUIStandalonePreset
 								],
-								layout: "BaseLayout"
+								layout: "BaseLayout",
+                ` + persistAuthString + `
```

Comment:
```suggestion
                ${persistAuthString}
```

Same here, should be more readable.